run: +error_task
+error_task:
  error:
    py>: examples.error_task.show_error

  +test:
    py>: examples.error_task.fails
